### DISCUSSION TITLE
Structure view - disable automatic expansion and keep everything collapsed?

### DISCUSSION THREAD

**Discussion by vsht** [texstudio-org/texstudio#1738] [Q&A] [answered]
Hi,

I'm working with documents that represent chapters of a book and hence  contain many sections, subsections and subsubsections.
This makes my structure view panel look very overloaded, since essentially 1-2 documents have "enough structure" to fill the whole space on a WQHD screen. 

I can collapse the structure view for each document by hand, but then when I switch to one of those documents via tabs,
the structure of the document get automatically expanded again. 

So I'm wondering if there is any way to disable this automatic expansion? That is, I would like to have the structure expanded only if I do it myself in the structure view but otherwise it should always remain collapsed.

Cheers,
Vladyslav

**Comment by foreachthing** [comment]
And that should be per document. IMHO. 
I like my docs expanded an then collaps them on demand.

**Reply by vsht** [reply]
Since I'm somewhat afraid that the devs might not understand my issue, here is a screenshot 
illustrating the problem. When you have multiple files with numerous sections and subsections (and perhaps even subsubsections),
the current behavior of automatically expanding everything in the structure view pane is really inconvenient...

![Bildschirmfoto vom 2021-08-26 17-18-09](https://user-images.githubusercontent.com/7283188/130990167-54382ba8-c55c-439d-a9f5-7f11c3ab5abd.jpg)

**Comment by sunderme** [comment, marked as answer]
4.0.0 will have an option to disable automatic structure scrolling, see #1742

**Reply by vsht** [reply]
Many thanks!

**Comment by sunderme** [comment]
by the way, this is actually a feature request. Best to report it as such (issue) as it keeps tracking simpler.

**Reply by vsht** [reply]
Ok, understood.
